src/test/java/de/tum/in/www1/artemis/service/compass/umlmodel/deployment/UMLDeploymentDiagrams.java

```diff
@@ -6,13 +6,13 @@
 
 import org.apache.commons.io.IOUtils;
 
-final class UMLDeploymentDiagrams {
+public final class UMLDeploymentDiagrams {
 
-    static final String DEPLOYMENT_MODEL_1;
+    public static final String DEPLOYMENT_MODEL_1;
 
-    static final String DEPLOYMENT_MODEL_2;
+    public static final String DEPLOYMENT_MODEL_2;
 
-    static final String DEPLOYMENT_MODEL_3;
+    public static final String DEPLOYMENT_MODEL_3;
```

Comment:
I probably forgot some. I wanted to make all test classes public (so that we have it consistently) and I think if the models are used in other test classes as well, it makes sense to make them public as well to have a better discoverability (which is a general problem in the tests, a lot of code is duplicated here or people write it again even if there is already existing code to solve some task during testing)